### ISSUE TITLE
Methods of evaluation in a CULane style custom dataset

### ISSUE THREAD

**Issue by thkimKETI** [cfzd/Ultra-Fast-Lane-Detection-v2#123]
I deeply appreciate your excellent work. 

I have created a custom dataset in the style of CULane and intend to train it. 

I have completed the installation of the culane evaluation tool by running the make file.

However, the current code seems to be loading the test dataset. I'm curious about how to perform validation.

In other words, I would like to conduct evaluations such as F-measure values for the validation dataset.

**Comment by cfzd** [comment]
@thkimKETI 
You can simply replace the loading of test set to the validation set. 
https://github.com/cfzd/Ultra-Fast-Lane-Detection-v2/blob/c903880678454dfd9b55a63022368db05c00bc6d/evaluation/eval_wrapper.py#L502

**Comment by yspark98** [comment]
@thkimKETI

hello, i'm park in inha university.
I have a question regarding your custom dataset.
Please your e-mail address or email to me yspark98@inha.edu

thank you.
